src/NServiceBus.Core/MessageMutators/Mutators.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NServiceBus.Features
+{
+    class Mutators : Feature
+    {
+        public Mutators()
+        {
+            EnableByDefault();
+        }
+
+        protected internal override void Setup(FeatureConfigurationContext context)
+        {
+            context.Pipeline.Register("MutateIncomingTransportMessage", new MutateIncomingTransportMessageBehavior(), "Executes IMutateIncomingTransportMessages");
```

Comment:
By moving the incoming mutator registrations out of `ReceiveFeature`, I was wondering how that would impact SendOnly endpoints, but then I realized that `ReceiveFeature` runs even when and endpoint is SendOnly. I suppose that means there's no problem here, but now I wonder why `ReceiveFeature` isn't set up with a prerequisite around SendOnly.